traveler-swift-core/TravelerKit/Models/BookingError.swift

```diff
@@ -17,15 +17,15 @@ public enum BookingError: Error {
     /// The `BookingOption` is required and was not supplied
     case noOption
     /// The requested number of passes is not available
-    case noPasses
+    case noPasses(String)
```

Comment:
Please add a label to this associated value, this way people can know what it refers to ---> noPasses(traceId: String). 

Also do it for the rest of the cases that use the traceId

Reply:
This change has been made. 
PR has been updated.